ports/openvino/vcpkg.json

```
@@ -14,7 +14,7 @@
   "homepage": "https://github.com/openvinotoolkit/openvino",
   "documentation": "https://docs.openvino.ai/latest/index.html",
   "license": "Apache-2.0",
-  "supports": "!uwp & !x86",
+  "supports": "!uwp & !x86 & !(android & arm)",
```

```suggestion
  "supports": "!uwp & !x86 & !(android & arm32)",
```

IIUC arm64 isn't affected.

---

Oh, you're right. I assumed the keyword would be arm for non arm64